woocommerce/woocommerce
# Fixed "virtual" and "downlodable" pointers on product walkthrough

### All Submissions:






### Changes proposed in this Pull Request:



On the product walkthrough the "Virtual products" and "Downlodable products" pointers aren't in the correct place:

<img width="889" alt="ScreenCapture at Tue Mar 24 15:31:56 EDT 2020" src="https://user-images.githubusercontent.com/2098816/77471328-74f0cd80-6de8-11ea-9dd8-cda7a4501ca3.png">

I fixed it by changing the location of the pointer, see:

![Screenshot from 2020-07-27 22-42-53](https://user-images.githubusercontent.com/1264099/88609805-6860d800-d05b-11ea-8b76-ecda99add133.png)

Closes #26006.

### How to test the changes in this Pull Request:

1. Go to `wp-admin/post-new.php?post_type=product&tutorial=true`
2. Go through the pointers until you find the "Virtual products" pointer, and check that isn't aligned with the checkbox.
3. Apply this patch and check the pointer again.

### Other information:




### Changelog entry

> Fix - Fixed "virtual" and "downlodable" pointers on product walkthrough.
